weave/ops_domain/run_history/history_op_common.py

```diff
@@ -110,22 +117,103 @@ def get_top_level_keys(key_tree: KeyTree) -> list[str]:
 def get_full_columns(columns: typing.Optional[list[str]]):
     if columns is None:
         return None
-    return list(set([*columns, "_step", "_runtime", "_timestamp"]))
+    return list(set([*columns, "_step"]))
```

Comment:
aren't `_runtime` and `_timestamp` still going to be present for history1&2?

Reply:
yes, but `_step` is the only key that internal resolver logic depends on. The others are fine to be selected by the user if desired